spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_environments.rb

```diff
@@ -56,5 +56,10 @@ def orchestration_supported?
       # We support orchestration from Havana
       environment_release_number >= 4
     end
+
+    def volume_snapshot_pagination_bug
+      # Seems like volume snapshot pagination in Liberty and below doesn't work
```

Comment:
Didn't work in Liberty?  Or, didn't work before Liberty?

I think this should read:

```ruby
 # Seems like volume snapshot pagination didn't work before Liberty
```

Is that right?